Python/fileutils.c

```diff
@@ -194,7 +194,13 @@ encode_ascii_surrogateescape(const wchar_t *text, size_t *error_pos)
 
     len = wcslen(text);
 
-    result = PyMem_Malloc(len + 1);  /* +1 for NUL byte */
+    /* +1 for NUL byte */
```

Comment:
Not related to your change but should it be "NULL byte" ?

Reply:
Oops, I only saw your review after I merged my PR. Anyway, I fixed this typo in my following PR: PR #4963.